packages/apps/desktop-viewer-test/src/frontend/hooks/useDownload.tsx

```diff
@@ -31,7 +31,7 @@ export const useDownload = (
     if (fileName) {
       const progressCallback: OnDownloadProgress = (progress) => {
         const { loaded, total } = progress;
-        const percent = loaded / total;
+        const percent = (loaded / total) * 100;
```

Comment:
good catch, but not the only place this changes needs to be done

Reply:
ah good catch, will add it to the pullChanges as well